airbyte-integrations/connectors/source-intercom/source_intercom/source.py

```diff
@@ -2,17 +2,536 @@
 # Copyright (c) 2023 Airbyte, Inc., all rights reserved.
```

Comment:
this file was reset by reverting [this PR](https://github.com/airbytehq/airbyte/pull/23013)